Fold result visualization docs in sidebar (#816)
grafana/k6-docs: src/data/markdown/translated-guides/en/01 Getting started/04 Results Output.md

excerpt: 'For basic tests, the top-level summary that k6 provides might be enough. For detailed analysis, you can stream all data your test outputs to an external source.'
---

As k6 generates load for your test, it also makes _metrics_ that measure the performance of the system.
Broadly, you can analyze metrics in two ways:
- As summary statistics, in an _end-of-test_ summary report.

> - In granular detail, with measurements for every point of the test (and timestamps)

Author: ```suggestion
- In granular detail, with measurements for every data point across test (and timestamps)
```
